lighthouse-cli/test/smokehouse/smokehouse.js

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/env node
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const path = require('path');
+const execSync = require('child_process').execSync;
+const yargs = require('yargs');
+
+const DEFAULT_CONFIG_PATH = 'pwa-config';
+const DEFAULT_EXPECTATIONS_PATH = 'pwa-expectations';
+
+const GREEN = '\x1B[32m';
+const RED = '\x1B[31m';
+const RESET = '\x1B[0m';
+const GREEN_CHECK = greenify('✓');
+const RED_X = redify('✘');
+
```

Comment:
> Do we have a file for constants like this just yet?

No, and I copied these from yet another file :) I personally am not a fan of central util files or whatever, as that's where random junk code goes to live forever.